### ISSUE TITLE
Throws an exception when used with Fastboot

### ISSUE THREAD

**Issue by aexmachina** [sir-dunxalot/ember-tooltips#130]
The call to `this.$()` doesn't return an Element when running in Fastboot:

```js
  target: computed(function () {
      var parentElement = this.$().parent();
```

Not sure what the right solution is, but maybe `if (!this.$()) return` would do the trick.

It doesn't appear to prevent the page from rendering - just throws an Exception.

**Comment by andrew-zenefits** [comment]
which component are you trying to use that in?

lazy-render-wrapper, tooltip-on-element, tooltip-on-component, popover-on-element, and popover-on-component are all tagless components.

**Comment by aexmachina** [comment]
Using `tooltip-on-element`.

**Comment by aexmachina** [comment]
I've submitted a PR.

**Comment by a15n** [comment]
thanks for submitting this PR. I've never worked with fastboot before so this is really appreciated. Is there a way we can add tests to for this? I don't know how we would. Right now we only have test coverage for different version of ember.

**Comment by aexmachina** [comment]
I've had a bit of a look and it's not clear. Basically you'd need to remove window, $ and change Component#$ to return null, and then run your whole test suite. It should be available as a test environment for Testem (same as Chrome or Firefox) but this hasn't been done. 

Basically we all need to write code that handles the DOM not being there outside of the Component hooks, which is what I've done in this PR. I reckon this is good to merge.

**Comment by sir-dunxalot** [comment]
Fixed by #131.
